driver/product/run_JGDAS_ATMOS_GEMPAK_META_NCDC_wcoss2.sh_00

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+#PBS -N gdas_atmos_gempak_meta_ncdc_00
+#PBS -j oe
+#PBS -l place=vscatter,select=4:ncpus=32:mpiprocs=32:mem=5GB
+#PBS -q debug
+#PBS -l walltime=00:30:00
+#PBS -A GFS-DEV
+#PBS -V
+#PBS -W umask=022
+
+cd $PBS_O_WORKDIR
+
+export PDY=`date -u +%Y%m%d`
+export PDY=20210824
+# export PDY=20210922
+
+export PDY1=`expr $PDY - 1`
+
+export cyc=00
+export cycle=t${cyc}z
+
+export threads=1
+export MP_LABELIO=yes
+
+set -xa
+export PS4='$SECONDS + '
+date
+
+#############################################################
+# Specify GFS versions
+#############################################################
+export gfs_ver=v16.0.0
```

Comment:
The GFS will be `v16.2.0` once in ops on WCOSS2.

Reply:
Thanks, Kate. I will update once GFS GFS will be v16.2.0 once in ops on WCOSS2.